src/static/helpers/environment.ts

```diff
@@ -24,20 +24,14 @@ export const globalObject = isBrowser ? window : globalThis;
 
 export const hasFetchAvailable = typeof globalObject.fetch === 'function';
 
+// TODO: Remove this function in the next major version
 // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 export const fetch: FetchFunction = (() => {
-  if (isNode) {
-    // .default is added because the newer versions of babel doesn't get the default export automatically for require().
-    // eslint-disable-next-line global-require, @typescript-eslint/no-unsafe-return, @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-member-access
-    return require('node-fetch').default;
-  }
-
-  // difficult to test in node environment
+  // Difficult to test in node environment
   /* istanbul ignore next */
-  if (!hasFetchAvailable)
-    throw new Error(
-      'Bad environment: it is not a node environment but fetch is not defined'
-    );
+  if (!hasFetchAvailable) {
+    throw new Error('Please use Node.js 18+ for native fetch support.');
```

Comment:
Should we still be asking folks to use 18+ instead of 20 given this is for avoiding breaking changes but we technically don't support Node 18 ?

Reply:
Node 18 is the first node version that exposes a native fetch function to node. I used node 18 instead of node 20/22 so that we don't have to manually update this error message every time we drop support for an older node version

If you feel strongly about it I can update it however